test-apps/display-test-app/src/frontend/RealityModelDisplaySettingsWidget.ts

```diff
@@ -0,0 +1,261 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import {
+  DisplayStyleSettings, FeatureAppearance, FeatureAppearanceProps, PointCloudDisplayProps, RealityModelDisplayProps, RealityModelDisplaySettings,
+} from "@itwin/core-common";
+import {
+  ContextRealityModelState, IModelApp, SpatialModelState, Tool, Viewport,
+} from "@itwin/core-frontend";
+import {
+  convertHexToRgb, createCheckBox, createColorInput, createRadioBox, createSlider,
+} from "@itwin/frontend-devtools";
+import { Surface } from "./Surface";
+import { Window } from "./Window";
+
+interface RealityModel {
+  readonly name: string;
+  settings: RealityModelDisplaySettings;
+  appearance: FeatureAppearance | undefined;
+}
+
+class ContextModel implements RealityModel {
+  private readonly _state: ContextRealityModelState;
+
+  public constructor(state: ContextRealityModelState) {
```

Comment:
Could use constructor initialization shorthand here/throughout this file - `public constructor(private readonly _state: ContextRealityModelState)`

Reply:
I absolutely despise that and delete it wherever I come across it in our code.

Reply:
Understood - saw it used in some places in the repo so wasn't sure if there was an opinion on it or not.